pom.xml

```diff
@@ -152,6 +152,8 @@
         <air.check.skip-jacoco>true</air.check.skip-jacoco>
         <air.java.version>22.0.0</air.java.version>
         <air.javadoc.lint>syntax,reference,html,-missing</air.javadoc.lint>
+        <!-- require at least maven 3.9.8 or mvnd 1.0.0+ -->
+        <air.maven.version>3.9.8</air.maven.version>
```

Comment:
airbase defaults air.maven.version to 3.6.3
what's the benefit of setting this differently in trino?

Reply:
I can set this in airbase

Reply:
https://github.com/airlift/airbase/pull/414